nav-app/src/app/config.service.ts

```diff
@@ -18,6 +18,9 @@ export class ConfigService {
             //parse feature preferences from config json
             config["features"].forEach(function(featureObject: any) {
                 self.setFeature_object(featureObject);
+                if (featureObject.name === "sub-techniques") {
+                    dataService.subtechniquesEnabled = featureObject.enabled;
+                }
```

Comment:
This methodology causes a bug. 

There are two ways the user can change the feature-set of the Navigator: 
- Modify the structure in config.json manually.
- create a customized navigator link using the "create customized navigator" section on a new tab. When the Navigator loads it parses the query string (technically the fragment, but we use it like a query string) to set the initial state of the structure.

The options in the create customized navigator menu are automatically generated from the features structure and assuming the API is used properly there is no difference between the two options. By setting the controls in the code here, instead of using the API for checking if a feature is set, the user can't enable/disable navigator sub-techniques from the query string method.

Unfortunately it may not be possible to use the API as intended because that would make a dependency loop (config.service would be codependent on data.service, which IIRC is difficult to do).

This should be fixable by putting the dataService.subtechniquesEnabled setter to after the fragments have been parsed, and use `self.getFeature()` to get/set the value.